app/pathogen/arbovirus/dashboard/(visualizations)/country-seroprevalence-comparison-scatter-plot.tsx

```diff
@@ -15,7 +15,7 @@ import uniq from "lodash/uniq";
 import { typedObjectKeys } from "@/lib/utils";
 import { ContentType } from "recharts/types/component/Tooltip";
 import { useChartArbovirusDropdown } from "./chart-arbovirus-dropdown";
-import { ArboContext } from "@/contexts/pathogen-context/pathogen-contexts/arbo-context";
+import { ArboContext, ArbovirusEstimate } from "@/contexts/pathogen-context/pathogen-contexts/arbo-context";
```

Comment:
Confusing diff, will point out actual changes. This code is to make the green data point happen but there's a couple changes that don't really make a difference (stuff like declaring a variable earlier, indent, etc.) that the diff flags :/